src/app/containers/MediaPageMain/blocks/image.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { StoryPromoImage } from '../../StoryPromo/index.jsx'
+
+// eslint-disable-next-line react/prop-types
+const HeadingContainer = ({ uuid, path, height, width }) => {
```

Comment:
```suggestion
const ImageContainer = ({ uuid, path, height, width }) => {
```

Reply:
lol completely forgot to change those, gotta love copy and paste